src/Symfony/Component/Messenger/Transport/Receiver/ReceiverInterface.php

```diff
@@ -23,11 +27,31 @@ interface ReceiverInterface
      *
      * The handler will have, as argument, the received {@link \Symfony\Component\Messenger\Envelope} containing the message.
      * Note that this envelope can be `null` if the timeout to receive something has expired.
+     *
+     * If the received message cannot be decoded, the MessageDecodingFailedException
+     * should be caught, the message should be removed from the queue,
```

Comment:
Using the word “queue” assumes a specific transport. A transport might talk to a different data structure.